src/Symfony/Component/Translation/Translator.php

```diff
@@ -174,6 +202,13 @@ public function trans(?string $id, array $parameters = [], ?string $domain = nul
             }
         }
 
+        if ($this->globalTranslatableParameters && !isset($this->globalParameters[$locale])) {
+            $this->globalParameters[$locale] = []; // Avoid infinite loops
+            $this->globalParameters[$locale] = array_map(fn ($parameter) => $parameter->trans($this, $locale), $this->globalTranslatableParameters);
```

Comment:
Given that `$parameters` below has support for TranslatableInterface, do we actually need to handle those global parameters early ?